app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -54,8 +54,10 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
     try {
       buildPushMessage =
           BuildPushMessage.fromJson(json.decode(String.fromCharCodes(base64.decode(data))) as Map<String, dynamic>);
+      log.info('Result message from base64: ${String.fromCharCodes(base64.decode(data))}');
```

Comment:
Nit: define a variable for `String.fromCharCodes(base64.decode(data))` to avoid duplicate operations.

Reply:
Done!